gluten-ut/src/test/scala/io/glutenproject/utils/velox/VeloxNotSupport.scala

```diff
@@ -42,6 +42,7 @@ object VeloxNotSupport extends NotSupport {
     simpleClassName[StringExpressionsSuite],
     simpleClassName[MiscExpressionsSuite],
     simpleClassName[NondeterministicSuite],
-    simpleClassName[RandomSuite]
+    simpleClassName[RandomSuite],
+    simpleClassName[ConditionalExpressionSuite]
```

Comment:
There is a blackTestNameList in GlutenConditionalExpressionSuite that was brought by PR#599, please see https://github.com/oap-project/gluten/blob/main/gluten-ut/src/test/scala/org/apache/spark/sql/catalyst/expressions/GlutenConditionalExpressionSuite.scala#L26. Would you remove that blacklist?

Reply:
I try to remove the blacklist and it seems can pass the ut in my local env.

Reply:
LGTM